mirrord/cli/src/container/command_builder.rs

```diff
@@ -57,19 +61,28 @@ impl RuntimeCommandBuilder {
         }
     }
 
-    pub fn add_volume<H, C>(&mut self, host_path: H, container_path: C)
+    pub fn add_volume<H, C>(&mut self, host_path: H, container_path: C, readonly: bool)
```

Comment:
Is `readonly` something that can be controlled by the user?

If it's not, then I would suggest changing it from a param to a const generic `<const READONLY: bool = true>` on this function.